cli/cliui/cliui.go

```diff
@@ -37,22 +43,33 @@ var (
 )
 
 var (
-	Green   = Color("#04B575")
-	Red     = Color("#ED567A")
-	Fuchsia = Color("#EE6FF8")
-	Yellow  = Color("#ECFD65")
-	Blue    = Color("#5000ff")
+	red           = Color("1")
+	green         = Color("2")
+	yellow        = Color("3")
+	magenta       = Color("5")
+	white         = Color("7")
+	brightBlue    = Color("12")
+	brightMagenta = Color("13")
 )
 
 // Color returns a color for the given string.
 func Color(s string) termenv.Color {
 	colorOnce.Do(func() {
 		color = termenv.NewOutput(os.Stdout).ColorProfile()
+
 		if flag.Lookup("test.v") != nil {
 			// Use a consistent colorless profile in tests so that results
 			// are deterministic.
 			color = termenv.Ascii
 		}
+
+		// Currently it appears there is no way to use the flag from
+		// serpent as it isn't possible to create a root middleware that
+		// runs for every command. For now we just check if `os.Args`
+		// has the flag.
+		if slices.Contains(os.Args, fmt.Sprintf("--%s", NoColorFlag)) {
+			color = termenv.Ascii
+		}
```

Comment:
Unfortunately not as `NoColor` is set _after_ our code runs. This is because there is no (apparent) way to have this initialization code ran before _every_ command but after serpent sets up the options.